### PR TITLE
[chrome_devtools] add Basic Auth example in Java

### PR DESCRIPTION
**Pull request by dbudim**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Added Java example of using Basic Auth via chrome dev tools

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/support_packages/chrome_devtools.de.md [hunk 1]
```diff
@@ -123,7 +123,10 @@ With Selenium and devtools integration, you can automate the input of basic auth
 
 {{< code-tab >}}
   {{< code-panel language="java" >}}
-# Please raise a PR to add code sample
+Predicate<URI> uriPredicate = uri -> uri.getHost().contains("your-domain.com");
+
+driver.register(uriPredicate, UsernameAndPassword.of("admin", "password"));
```
Comment (harsha509) [inline docs_source_files/content/support_packages/chrome_devtools.de.md]: Is this should not be in this way?

((HasAuthentication) driver).register(uriPredicate, UsernameAndPassword.of("admin", "password"));

**Review by harsha509 — COMMENTED** [review]
Hi @dbudim ,

Thank for the PR. 
Can you look into the comments.

Also, am unable to execute the code sample after doing modifications as below 

`         Predicate<URI> uriPredicate = uri -> uri.getHost().contains("https://the-internet.herokuapp.com");
            ((HasAuthentication) driver).register(uriPredicate, UsernameAndPassword.of("admin", "admin"));
            driver.get("https://the-internet.herokuapp.com/basic_auth");`

:wave @diemol , Can you please review the code sample and suggest the correct one.

Thank you!

File: docs_source_files/content/support_packages/chrome_devtools.de.md (hunk 1 above)
Comment (dbudim) [reply docs_source_files/content/support_packages/chrome_devtools.de.md]: Hi @JS31096,

Thanks for your comment)
You right, it's not clear that driver should implements HasAuthentication. 
I've used `ChromiumDriver driver = new ChromeDriver()` while tested previous version and didn't mind that it's not clear.

I've added cast to HasAuthentication.
Please approve)

**Review by harsha509 — APPROVED** [review]
Thank you @dbudim !

**Comment by harsha509** [comment]
Congratulations on your first contribution to Selenium docs @dbudim 🎉 🎉

**Comment by dbudim** [comment]
> Congratulations on your first contribution to Selenium docs @dbudim 🎉 🎉

Thank you  🎉 🎉 🎉

**Pull request merged**